kroxylicious-integration-tests/src/test/java/io/kroxylicious/proxy/filter/authorization/OffsetFetchAuthzIT.java

```diff
@@ -0,0 +1,280 @@
+                allowAllOnGroup(ALICE, GROUP_ID),
+                allowAllOnGroup(BOB, GROUP_ID),
+                allowAllOnGroup(EVE, GROUP_ID));
+    }
+
+    private static AclBinding allowAllOnGroup(String user, String groupId) {
+        return new AclBinding(
+                new ResourcePattern(ResourceType.GROUP, groupId, PatternType.LITERAL),
+                new AccessControlEntry("User:" + user, "*",
+                        AclOperation.ALL, AclPermissionType.ALLOW));
+    }
+
+    @BeforeEach
+    void prepClusters() {
+        this.topicIdsInUnproxiedCluster = prepCluster(kafkaClusterWithAuthz, ALL_TOPIC_NAMES_IN_TEST, aclBindings);
+        produceAndConsumeToEstablishGroup(GROUP_ID, kafkaClusterWithAuthz);
+        this.topicIdsInProxiedCluster = prepCluster(kafkaClusterNoAuthz, ALL_TOPIC_NAMES_IN_TEST, List.of());
+        produceAndConsumeToEstablishGroup(GROUP_ID, kafkaClusterNoAuthz);
+    }
+
+    private static void produceAndConsumeToEstablishGroup(String groupId, KafkaCluster cluster) {
+        Map<String, Object> producerConfig = cluster.getKafkaClientConfiguration(SUPER, "Super");
+        producerConfig.put(ProducerConfig.KEY_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+        producerConfig.put(ProducerConfig.VALUE_SERIALIZER_CLASS_CONFIG, StringSerializer.class);
+
+        Map<String, Object> consumerConfig = cluster.getKafkaClientConfiguration(SUPER, "Super");
+        consumerConfig.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        consumerConfig.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
+        consumerConfig.put(ConsumerConfig.GROUP_ID_CONFIG, groupId);
+        consumerConfig.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest");
+
+        try (CloseableProducer<Object, Object> producer = new CloseableProducer<>(
+                new KafkaProducer<>(producerConfig));
+                CloseableConsumer<Object, Object> aSuper = new CloseableConsumer<>(new KafkaConsumer<>(consumerConfig))) {
+            aSuper.subscribe(ALL_TOPIC_NAMES_IN_TEST);
+            for (String topicName : ALL_TOPIC_NAMES_IN_TEST) {
+                producer.send(new ProducerRecord<>(topicName, "value"));
+            }
+            producer.flush();
+            int observedRecords = 0;
+            while (observedRecords < ALL_TOPIC_NAMES_IN_TEST.size()) {
+                ConsumerRecords<Object, Object> consumerRecords = aSuper.poll(Duration.ofMillis(50));
+                observedRecords += consumerRecords.count();
+            }
+        }
+        catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    @AfterEach
+    void tidyClusters() {
+        deleteTopicsAndAcls(kafkaClusterWithAuthz, ALL_TOPIC_NAMES_IN_TEST, aclBindings);
+        deleteTopicsAndAcls(kafkaClusterNoAuthz, ALL_TOPIC_NAMES_IN_TEST, List.of());
+    }
+
+    List<Arguments> test() {
```

Comment:
Probably we should be naming the tests better than just test. Especially for APIs which apply both topic and group authz. We'll eventually need both `shouldEnforceAccessToTopics` and `shouldEnforceAccessToGroups`.